### DISCUSSION TITLE
Edit on the current page

### DISCUSSION THREAD

**Discussion by rebeleder** [gsantner/markor#1463] [Ideas]
When editing in view mode, I hope that I can directly position the cursor to the progress I am currently viewing.

**Comment by gsantner** [comment]
There is no editing in view mode.

**Reply by rebeleder** [reply]
Sorry, I didn't express my meaning clearly. What I mean is, for example, when I click edit while viewing the middle part of the document, the edited view will be on the top instead of the middle part.

**Reply by gsantner** [reply]
What you probably look for is `https://github.com/gsantner/markor/issues/679`, which is unlikely to become available.

**Reply by rebeleder** [reply]
I get it，thanks
